Gopkg.toml

```diff
@@ -19,7 +19,7 @@ required = [
 
 [[constraint]]
   name = "github.com/knative/build"
-  version = "v0.2.0"
+  version = "v0.3.0"
```

Comment:
I think we also need to replace the release yaml in `third_party/config/build`

Reply:
Good point !